.github/workflows/ci.yml

```diff
@@ -63,7 +86,7 @@ jobs:
         if: matrix.test_type == 'release-upgrade' || matrix.test_type == 'source-upgrade'
         run: |
           docker pull quay.io/pulp/pulp-ci-dbuster:3.0.0
-          docker pull quay.io/pulp/pulp_rpm-ci-f31:3.1.0
```

Comment:
A side effect of having these configs copied in 3 different places - we forget to update things everywhere.

Reply:
Thank you for updating it!